src/utils/Xhr.js

```diff
@@ -118,10 +126,17 @@ class Xhr {
             return false;
         }
 
-        const { response, request } = error;
+        // Network methods that are safe to retry
+        const idempotentMethods = [HTTP_GET, HTTP_OPTIONS, HTTP_HEAD].map(s => s.toLowerCase());
+
+        const { response, request, config } = error;
         // Retry if there is a network error (e.g. ECONNRESET) or rate limited
+        const status = getProp(response, 'status');
+        const method = getProp(config, 'method');
         const isNetworkError = request && !response;
-        return isNetworkError || getProp(response, 'status') === HTTP_STATUS_CODE_RATE_LIMIT;
+        const isRateLimitError = status === HTTP_STATUS_CODE_RATE_LIMIT;
+        const isOtherRetryableError = this.retryableStatusCodes.includes(status) && idempotentMethods.includes(method);
```

Comment:
It looks like `.includes` isn't supported natively in IE11. May as well swap it out for a lodash version.